### ISSUE TITLE
Feat:FEEDBACK SECTION

### ISSUE THREAD

**Issue by IpsitaPrusty** [Its-Aman-Yadav/Community-Site#254]
### Describe the feature

When we tap on feedback, it doesn't work. so by adding a page where there will be credentials to be asked by the user and ask them about their experience. Is it worth of information's or not etc.

please @Its-Aman-Yadav sir, assign this to me.

### Add ScreenShots

<img width="945" alt="fff" src="https://github.com/Its-Aman-Yadav/Community-Site/assets/145246053/57281f44-b0ec-4b92-a15d-1c60d87c56e4">
It doesn't do anything.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by yash5080** [comment]
assign me this task

**Comment by Its-Aman-Yadav** [comment]
Yeah sure, please work on it @IpsitaPrusty

**Comment by PavanKumarNaik936** [comment]
yes, the feedback web page is not working there.  **@Its-Aman-Yadav** could you please assign to me sir ,so that i can work on this by adding user-friendly feedback page with required information

**Comment by Siddhi561** [comment]
Hey, can you please assign me this issue if it is not solved? This is my first time making an open-source contribution, and I really want to showcase my skills. I am very familiar with languages like Java, HTML, CSS, and JS. Apart from coding, I'm an ux/ui freelancer as well.

**Comment by IpsitaPrusty** [comment]
> yes, the feedback web page is not working there. **@Its-Aman-Yadav** could you please assign to me sir ,so that i can work on this by adding user-friendly feedback page with required information

its about to be complete.

**Comment by IpsitaPrusty** [comment]
> Hey, can you please assign me this issue if it is not solved? This is my first time making an open-source contribution, and I really want to showcase my skills. I am very familiar with languages like Java, HTML, CSS, and JS. Apart from coding, I'm an ux/ui freelancer as well.

Its about to be complete. Also its my first time making an open-source contribution.

**Comment by princyym** [comment]
I would like to contribute to this issue. Could you assign it to me, @Its-Aman-Yadav?
